src/components/flat-table/flat-table-test.stories.tsx

```diff
@@ -789,3 +790,49 @@ export const FlatRowHeaderWithNoPaddingAndButtons = () => {
     </FlatTable>
   );
 };
+
+export const MinimalDesign = () => {
+  return (
+    <FlatTable colorTheme="transparent-white" hasNoHorizontalBorders={false}>
```

Comment:
comment: this isn't defined on the `FlatTable` interface, it's `horizontalBorders` that I think needs using here

Reply:
I'm a bit confused by this implementation overall though, the `horizontalBorders` says it will make them transparent but below we also have `horizontalBorderColor="--colorsUtilityMajor050"` etc set on all the body rows. I think a name like `hasVerticalBorders` would be the most intuitive name but `verticalBorders` would also work